rktl_sim/src/simulator/car.py

```diff
@@ -104,23 +124,54 @@ def get_pose(self, noise=None):
                 pos = np.random.normal(pos, noise['pos'])
                 orient = np.random.normal(orient, noise['orient'])
 
-        return (pos, p.getQuaternionFromEuler(orient))
+        return pos, p.getQuaternionFromEuler(orient)
 
     def get_velocity(self):
+        """
+        @return: the velocity of the car
+        """
```

Comment:
We can probably make this more informative. Right now it doesn't provide more information than the function name. Maybe detail on the units / framing.

Reply:
I totally agree that this should be more informative. I looked over the car's get_velocity function and it returns linear and angular velocity. Out of curiosity, what do the link_state[6:8] commands do, as I feel it would be a good thing to jot down.